src/databaseUtils.ts

```diff
@@ -27,3 +27,5 @@ export type ViewSchema = {
 export type TableNames = keyof Tables;
 
 export type ViewNames = keyof Views;
+
+export type ParcelStatus = Schema["status_order"]["event_name"];
```

Comment:
Not sure if this is the right place to put it

Reply:
This is good!